CRYOSMS/CRYOSMS-IOC-01App/src/build.mak

```diff
@@ -0,0 +1,76 @@
+TOP=../..
+
+include $(TOP)/configure/CONFIG
+#----------------------------------------
+#  ADD MACRO DEFINITIONS AFTER THIS LINE
+#=============================
+
+### NOTE: there should only be one build.mak for a given IOC family and this should be located in the ###-IOC-01 directory
+
+#=============================
+# Build the IOC application CRYOSMS-IOC-01
+# We actually use $(APPNAME) below so this file can be included by multiple IOCs
+
+PROD_IOC = $(APPNAME)
+# CRYOSMS-IOC-01.dbd will be created and installed
+DBD += $(APPNAME).dbd
+
+$(APPNAME)_DBD += base.dbd
+## ISIS standard dbd ##
+$(APPNAME)_DBD += icpconfig.dbd
+$(APPNAME)_DBD += pvdump.dbd
+$(APPNAME)_DBD += asSupport.dbd
+$(APPNAME)_DBD += devIocStats.dbd
+$(APPNAME)_DBD += caPutLog.dbd
+$(APPNAME)_DBD += utilities.dbd
+## add other dbd here ##
+$(APPNAME)_DBD += stream.dbd
+$(APPNAME)_DBD += asyn.dbd
+$(APPNAME)_DBD += drvAsynSerialPort.dbd
+$(APPNAME)_DBD += drvAsynIPPort.dbd
+$(APPNAME)_DBD += calcSupport.dbd
+$(APPNAME)_DBD += ReadASCII.dbd
+$(APPNAME)_DBD += FileList.dbd
+$(APPNAME)_DBD += asubFunctions.dbd
+$(APPNAME)_DBD += gsl.dbd
+$(APPNAME)_DBD += CRYOSMS.dbd
+
+# Add all the support libraries needed by this IOC
+## ISIS standard libraries ##
+$(APPNAME)_LIBS += seq pv
+$(APPNAME)_LIBS += devIocStats 
+$(APPNAME)_LIBS += pvdump $(MYSQLLIB) easySQLite sqlite 
+$(APPNAME)_LIBS += caPutLog
+$(APPNAME)_LIBS += icpconfig pugixml
+$(APPNAME)_LIBS += autosave
+## Add other libraries here ##
+$(APPNAME)_LIBS += stream
+$(APPNAME)_LIBS += ReadASCII
+$(APPNAME)_LIBS += FileList
+$(APPNAME)_LIBS += calc sscan
+$(APPNAME)_LIBS += utilities
+$(APPNAME)_LIBS += asyn
+$(APPNAME)_LIBS += pcre libjson zlib
+$(APPNAME)_LIBS += efsw
+$(APPNAME)_LIBS += asubFunctions gsl gslSupport
+$(APPNAME)_LIBS += cryosms
```

Comment:
There are a lot of additional dbds and libs in here that I don't think are required. https://github.com/ISISComputingGroup/EPICS-base/blob/master/src/template/base/top/iocApp/src/build.mak holds the list of the bare minimum ones that you need and I don't think you're using many of these others?